### PR TITLE
Refactored the portfolio workshop for Amy's code cadets

### PR DESCRIPTION
**Pull request by jonleung**
Not ready for merge yet.

### CODE REVIEW

**Comment by jonleung** [comment]
@zachlatta I made the grammar changes that Amy suggested!

**Comment by jonleung** [comment]
@zachlatta, note that the tests fail because I chose to omit `http` in my external links. This decision was made to improve readability.

**Comment by zachlatta** [comment]
I feel very strongly that omitting http/https worsens readability because you could be linking to any one of a variety of protocols it from the URL makes it ambiguous and confusing to the user. It also worsens functionality because if an https version of the site is available, we'd want to direct the user to it for the sake of privacy, but omitting the protocol entirely will send them to the http version (at least last time I checked the major browsers).

## 

Zach Latta
hackEDU| zach@hackedu.us

File: cohort_4/playbook/workshops/culture_tech_cadets_portfolio/README.md [hunk 1]
```diff
@@ -0,0 +1,490 @@
...
+- learn how to teach yourself new concepts in coding by using Google.
+
+# Setting up your tools
+
+Just like you may write your English essays in Microsoft Word, we will write
+Code in Cloud9.
+
+After you write your English essay, sometimes you save and share it on the
+internet with tools such as Dropbox. The place where many professional developers save
+and share their code is on a website called Github.com.
```
Comment (zachlatta) [inline cohort_4/playbook/workshops/culture_tech_cadets_portfolio/README.md]: It's GitHub, not Github

File: cohort_4/playbook/workshops/culture_tech_cadets_portfolio/README.md [hunk 2]
```diff
@@ -0,0 +1,490 @@
...
+
+In the following steps, we will be setting up your own Github and Cloud9
+accounts.
+
+## Open [`github.com`](http://github.com)
```
Comment (zachlatta) [inline cohort_4/playbook/workshops/culture_tech_cadets_portfolio/README.md]: Can you make this HTTPS?

File: cohort_4/playbook/workshops/culture_tech_cadets_portfolio/README.md [hunk 3]
```diff
@@ -0,0 +1,490 @@
...
+Here's what my `index.html` looks like
+
+```html
+<!DOCTYPE html>
+<head>
+  <style>
+    body {
+      text-align: center;
+      font-family: helvetica;
+      background: url("http://i.imgur.com/uVIuDll.jpg");
```
Comment (zachlatta) [inline cohort_4/playbook/workshops/culture_tech_cadets_portfolio/README.md]: Can you make this HTTPS?

File: cohort_4/playbook/workshops/culture_tech_cadets_portfolio/README.md [hunk 4]
```diff
@@ -0,0 +1,490 @@
...
+
+### So how might I figure this out?
+
+I might Google `html links`
+
+These are the Google search results I get:
+
+> ![](img/google_html_links.png)
+
+I click on the first result, ["HTML Links - W3Schools"](http://www.w3schools.com/html/html_links.asp)
```
Comment (zachlatta) [inline cohort_4/playbook/workshops/culture_tech_cadets_portfolio/README.md:385]: Can this be HTTPS?

File: cohort_4/playbook/workshops/culture_tech_cadets_portfolio/README.md (hunk 1 above)
Comment (jonleung) [inline cohort_4/playbook/workshops/culture_tech_cadets_portfolio/README.md]: Cool. I made a fix that you'll see in the next commit.

File: cohort_4/playbook/workshops/culture_tech_cadets_portfolio/README.md (hunk 4 above)
Comment (jonleung) [inline cohort_4/playbook/workshops/culture_tech_cadets_portfolio/README.md:385]: No, w3 schools does not support `https`

File: cohort_4/playbook/workshops/culture_tech_cadets_portfolio/README.md (hunk 3 above)
Comment (jonleung) [inline cohort_4/playbook/workshops/culture_tech_cadets_portfolio/README.md]: Cool. I made a fix that you'll see in the next commit.

File: cohort_4/playbook/workshops/culture_tech_cadets_portfolio/README.md (hunk 2 above)
Comment (jonleung) [inline cohort_4/playbook/workshops/culture_tech_cadets_portfolio/README.md]: Cool. I made a fix that you'll see in the next commit.

**Comment by jonleung** [comment]
@zachlatta, I made the fixes that you suggested. Thanks!

File: cohort_4/playbook/workshops/culture_tech_cadets_portfolio/README.md [hunk 5]
```diff
@@ -0,0 +1,490 @@
+# Building Your First Website
+
+In this tutorial, you will be learning how to build your first website!
+
+There are two parts of this tutorial:
+
+In part 1, we'll be using the website [Dash](dash.generalassemb.ly) to learn the
```
Comment (zachlatta) [inline cohort_4/playbook/workshops/culture_tech_cadets_portfolio/README.md]: You need to include the protocol for the link to work, else GitHub thinks you're linking to a file. This link is currently broken.

File: cohort_4/playbook/workshops/culture_tech_cadets_portfolio/README.md [hunk 6]
```diff
@@ -0,0 +1,490 @@
...
+fundamentals of building a website.
+
+Then in part 2, we'll be making our website using a tool called [Cloud9](c9.io).
```
Comment (zachlatta) [inline cohort_4/playbook/workshops/culture_tech_cadets_portfolio/README.md]: Same thing here. Broken link.

**Comment by maxwofford** [comment]
@jonleung Any updates on this?

File: cohort_4/playbook/workshops/culture_tech_cadets_portfolio/README.md (hunk 5 above)
Comment (jonleung) [inline cohort_4/playbook/workshops/culture_tech_cadets_portfolio/README.md]: Cool. I made a fix that you'll see in the next commit.

File: cohort_4/playbook/workshops/culture_tech_cadets_portfolio/README.md (hunk 6 above)
Comment (jonleung) [inline cohort_4/playbook/workshops/culture_tech_cadets_portfolio/README.md]: Cool. I made a fix that you'll see in the next commit.

**Comment by jonleung** [comment]
@MaxWofford and @zachlatta, I believe I have made the appropriate fixes to address the issues that @zachlatta brought up. The pull request should be ready to merge.

File: cohort_4/playbook/workshops/culture_tech_cadets_portfolio/README.md [hunk 7]
```diff
@@ -0,0 +1,490 @@
+# Building Your First Website
+
+In this tutorial, you will be learning how to build your first website!
+
+There are two parts of this tutorial:
+
+In part 1, we'll be using the website [Dash](https://dash.generalassemb.ly) to learn the
+fundamentals of building a website.
+
+Then in part 2, we'll be making our website using a tool called [Cloud9](c9.io).
```
Comment (maxwofford) [inline cohort_4/playbook/workshops/culture_tech_cadets_portfolio/README.md]: @jonleung The test failed because this requires https://c9.io

**Comment by jonleung** [comment]
@MaxWofford, I just made the fix. Thanks! @zachlatta, it should be ready to merge now I think?

**Comment by maxwofford** [comment]
screenshot 2015-08-25 at 3 56 52 pm
@jonleung Did you remember to commit your fix?

**Pull request merged**